test/passing/long_let_in.ml.ref

```diff
@@ -0,0 +1,59 @@
+[@@@ocamlformat "break-before-in=false"]
+
+let flat : unit =
+  let short = 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 in
+  let fooo =
+    1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 in
+  let baaar =
+    1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 in
+  let long =
+    1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1
+    in
```

Comment:
Having `in` alone and indented looks weird

Reply:
I made it unindented.

There is discussion to allow margin breaking for the corner case (along the lines of `break_before_in: [ ``Always | ``Only_to_save_margin | ``Never ]` with `Never` simply ignoring the margin for the purpose of laying the `in`) I might make another PR in the future.